app/views/responsiblepeople/legal_name.scala.html

```diff
@@ -32,13 +32,16 @@
 ) {
     @form(f, controllers.responsiblepeople.routes.LegalNameController.post(index, edit, flow)) {
 
+
+        <p>@Messages("responsiblepeople.legalnamechangedate.paragraph")</p>
```

Comment:
This is correct?

Reply:
I think so, this page was marked to do both paragraph and hint, so I did it :)

Reply:
Cool